pallets/subtensor/src/macros/errors.rs

```diff
@@ -150,10 +148,14 @@ mod errors {
         TooManyChildren,
         /// Default transaction rate limit exceeded.
         TxRateLimitExceeded,
-        /// Swap already scheduled.
```

Comment:
Shouldn't we deprecate enums instead of deleting them?

Reply:
@shamil-gadelshin I was under the impression the enum are safe to add/remove elements when they are not used in long term storage:
- errors are not in storage and are decoded correctly using the latest stored runtime code.
- events are only stored in the actual block the event is emitted, this is safe because events storage is never read directly
- enum stored in any other storage item are potentially unsafe to modify

I realized this after updating the proxy pallet one day and [an event was inserted before end and it was fine](https://github.com/paritytech/polkadot-sdk/pull/7995/files#diff-c434cd259ed280c05a395c61be631e464b9454a5a331e2ce393e196d091d837d)

Only case it breaks is if clients use hardcoded metadata/index instead of last one as @camfairchild said